### ISSUE TITLE
add naval support units

### ISSUE THREAD

**Issue by SGHQ2** [Anuken/Mindustry-Suggestions#1730]
there is just 1 naval units and that all. the naval support unit is a unit that can heal building only on the tier 1 and tier 2, the tier 2 to 5 however are the units that you can able to build onto the unit like a duo with the item source that moves just to make the naval units not boring. the tier 1 and tier 2 have there own weapons but tier 3 to 5 didn't so you just have to make a turret onto them so it will not do nothing like a oct. the tier 5 of the naval support unit have a shield that has the same health as a force projector and it can build risso units. when placing walls onto the unit it adds up to its health but the walls will destroy because of the damage taken, its like armor added.

**Comment by joshuaptfan** [comment]
There are [many naval support unit suggestions](is:issue naval support ), you could have commented on one of them instead of making a new suggestion.

If you had looked at the template instead of deleting it (you are supposed to fill it out), you would have seen that you were supposed to check this box:

> [ ] I have done a quick search in the list of suggestions to make sure this has not been suggested yet.

**Comment by Humanoid-X** [comment]
- [X] I AM INCAPABLE OF READING, COMPREHENDING AND COMPLETELY FILLING OUT THE TEMPLATE, BUT CAPABLE OF REMOVING IT ENTIRELY.

**Comment by itcannotbe** [comment]
>     * [x]  I AM INCAPABLE OF READING, COMPREHENDING AND COMPLETELY FILLING OUT THE TEMPLATE, BUT CAPABLE OF REMOVING IT ENTIRELY.

That line is oddly familiar.

**Comment by Bohaska** [comment]
<img width="407" alt="Screen Shot 2020-12-28 at 11 02 23 AM" src="https://user-images.githubusercontent.com/73286691/103186655-56ca7780-48fc-11eb-8153-cc8b9c9d4717.png">

**Comment by Humanoid-X** [comment]
I totally agree with @joshuaptfan . 
And yeah @itcannotbe I don't know what's happened to my brain, but this line sound oddly familiar too
LOLOL

**Comment by Kyllingene** [comment]
> That line is oddly familiar.

I believe the line is unnecessarily toxic. For one, it implies that the user is incapable of reading, comprehending instructions, and/or filling out forms, all of which insult the poster. For another, several people have had Github glitch and give them no templates, and then add the label themselves. In such a case, commending them for the label addition is the correct path, not ridiculing them for not using the template. Here are some quotes from the [Github Community Guidelines](https://docs.github.com/en/free-pro-team@latest/github/site-policy/github-community-guidelines):

> don’t post anything that a reasonable person would consider offensive

As long as you believe that I am being reasonable in this case, then the line could easily be considered offensive.

> Avoid name-calling, ad hominem attacks

It is a blatant ad hominem.

> Don’t harass or grief anyone

And even if it is not harassing on its own, it is most commonly followed up by more attacks.

TL;DR: Don't be rude, read the [guidelines](https://docs.github.com/en/free-pro-team@latest/github/site-policy/github-community-guidelines).

**Comment by itcannotbe** [comment]
TBH. I am just stating the facts the due to the lack of a filled out template or even the template itself, OP made a duplicate issue. There is nothing toxic about pointing that out.

**Comment by Kyllingene** [comment]
No, I agree that pointing it out is necessary. The method used - insulting the OP - is a bad method.

**Comment by Humanoid-X** [comment]
Alright, I will stop if it is a glitch. But, please, fill out the template that Anuke made, unless you want Anuke to ignore your suggestion.

**Comment by Kyllingene** [comment]
That, I agree with.
